src/components/EditorsColumn.jsx

```diff
@@ -13,58 +16,115 @@ function allErrorsFor(language, errors, runtimeErrors) {
+    this.editorRefs = new Map([[1, null], [2, null], [3, null]]);
+    bindAll(
+      this,
+      '_storeDividerRef',
+      '_storeEditorRef',
+      '_handleEditorDividerDrag',
+    );
+  }
 
-    editors.push(
-      <EditorContainer
-        key={language}
-        language={language}
-        source={currentProject.sources[language]}
-        onHide={
-          partial(onComponentHide, `editor.${language}`)
-        }
-      >
-        <Editor
-          errors={allErrorsFor(language, errors, runtimeErrors)}
+  _storeEditorRef(index, editor) {
+    this.editorRefs.set(index, editor);
+  }
+
+  _storeDividerRef(index, divider) {
+    this.dividerRefs.set(index, divider);
+  }
+
+  _handleEditorDividerDrag(index, _, {deltaY, lastY, y}) {
+    const {onUpdateFlex} = this.props;
+    const editorHeights = getNodeHeights(this.editorRefs);
+    const dividerHeights = getNodeHeights(this.dividerRefs);
+    onUpdateFlex({deltaY, dividerHeights, editorHeights, index, lastY, y});
+  }
+
+  render() {
+    const {
+      currentProject,
+      editorsFlex,
+      errors,
+      onComponentHide,
+      onEditorInput,
+      onRequestedLineFocused,
+      runtimeErrors,
+      ui,
+    } = this.props;
+
+    const children = [];
+    const languages = ['html', 'css', 'javascript'].filter(language =>
+      !includes(currentProject.hiddenUIComponents, `editor.${language}`),
+    );
+    languages.forEach((language, index) => {
+      children.push(
+        <EditorContainer
           key={language}
           language={language}
-          percentageOfHeight={1 / editors.length}
-          projectKey={currentProject.projectKey}
-          requestedFocusedLine={ui.editors.requestedFocusedLine}
           source={currentProject.sources[language]}
-          onInput={partial(onEditorInput, language)}
-          onRequestedLineFocused={onRequestedLineFocused}
-        />
-      </EditorContainer>,
-    );
-  });
+          style={{flex: editorsFlex[index]}}
+          onHide={partial(onComponentHide, `editor.${language}`)}
+          onRef={partial(this._storeEditorRef, index)}
+        >
+          <Editor
+            errors={allErrorsFor(language, errors, runtimeErrors)}
+            key={language}
+            language={language}
+            percentageOfHeight={1 / languages.length}
+            projectKey={currentProject.projectKey}
+            requestedFocusedLine={ui.editors.requestedFocusedLine}
+            source={currentProject.sources[language]}
+            onInput={partial(onEditorInput, language)}
+            onRequestedLineFocused={onRequestedLineFocused}
+          />
+        </EditorContainer>,
+      );
+      if (index < languages.length - 1) {
+        children.push(
+          <DraggableCore
+            key={`divider:${language}`}
+            ref={partial(this._storeDividerRef, index)}
+            onDrag={partial(this._handleEditorDividerDrag, index)}
+          >
+            <div className="editors__divider" />
```

Comment:
I think if we attach `_storeDividerRef` to this `<div>` rather than to the `<DraggableCore>`, we should be able to avoid using `ReactDOM.findDOMNode()` altogether?